test/powershell/Language/Parser/ExtensibleCompletion.Tests.ps1

```diff
@@ -355,3 +355,97 @@ Describe "Additional type name completion tests" -Tags "CI" {
         TestInput = 'Get-Command -ParameterType System.Collections.Generic.Dic'
     } | Get-CompletionTestCaseData | Test-Completions
 }
+
+Describe "ArgumentCompletionsAttribute tests" -Tags "CI" {
+
+    BeforeAll {
+        function TestArgumentCompletionsAttribute
+        {
+            param(
+                [ArgumentCompletions("value1", "value2", "value3")]
+                $Alpha,
+                $Beta
+            )
+        }
+
+        function TestArgumentCompletionsAttribute1
+        {
+            param(
+                [ArgumentCompletionsAttribute("value1", "value2", "value3")]
+                $Alpha,
+                $Beta
+            )
+        }
+
+        $cmdletSrc=@'
+        using System;
+        using System.Management.Automation;
+        using System.Collections.Generic;
+
+        namespace Test.A {
+
+            [Cmdlet(VerbsCommon.Get, "ArgumentCompletions")]
+            public class TestArgumentCompletionsAttributeCommand : PSCmdlet
+            {
+                [Parameter]
+                [ArgumentCompletions("value1", "value2", "value3")]
+                public string Param1;
+
+                protected override void EndProcessing()
+                {
+                    WriteObject(Param1);
+                }
+            }
+
+            [Cmdlet(VerbsCommon.Get, "ArgumentCompletions1")]
+            public class TestArgumentCompletionsAttributeCommand1 : PSCmdlet
+            {
+                [Parameter]
+                [ArgumentCompletionsAttribute("value1", "value2", "value3")]
+                public string Param1;
+
+                protected override void EndProcessing()
+                {
+                    WriteObject(Param1);
+                }
+            }
+        }
+'@
+        $cls = Add-Type -TypeDefinition $cmdletSrc -PassThru | Select-Object -First 1
+        $testModule = Import-Module $cls.Assembly -PassThru
+
+        $testCasesScript = @(
+            @{ attributeName = "ArgumentCompletions"         ; cmdletName = "TestArgumentCompletionsAttribute"  },
+            @{ attributeName = "ArgumentCompletionsAttribute"; cmdletName = "TestArgumentCompletionsAttribute1" }
+        )
+
+        $testCasesCSharp = @(
+            @{ attributeName = "ArgumentCompletions"         ; cmdletName = "Get-ArgumentCompletions"  },
+            @{ attributeName = "ArgumentCompletionsAttribute"; cmdletName = "Get-ArgumentCompletions1" }
+        )
+    }
+
+    AfterAll {
+        #Remove-Module -ModuleInfo $testModule

```

Comment:
Did you forget to uncomment this?

Reply:
Yes 😄 Today test is passed. I wonder - I don't change a code.